test/e2e/egress_traffic_test.go

```diff
@@ -0,0 +1,66 @@
+package e2e
+
+import (
+	"encoding/base64"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/knative/serving/test"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	targetHostDomain = "httpbin.org"
+	testString       = "Would you like a cup of coffee?"
+)
+
+func TestEgressTraffic(t *testing.T) {
+	t.Parallel()
+	clients := Setup(t)
+	t.Log("Creating a Route and Configuration for httpproxy test app.")
+
+	svcName := test.ObjectNameForTest(t)
+	httpProxyNames := test.ResourceNames{
+		Config: svcName,
+		Route:  svcName,
+		Image:  "httpproxy",
+	}
+
+	envVars := []corev1.EnvVar{{
+		Name:  targetHostEnv,
+		Value: targetHostDomain,
+	}}
+	// Set up httpproxy app
+	t.Log("Creating a Route and Configuration for httpproxy test app.")
+	httpProxyNames, err := CreateRouteAndConfig(t, clients, "httpproxy", &test.Options{
```

Comment:
Why not `test.CreateRunLatestServiceReady`? You can then remove most everything up until line 56.